_utils/adapt.py

```diff
@@ -186,59 +184,75 @@ def __init__(self):
         if not self._instance:
             pass
 
+    # noinspection PyMethodParameters
     @property
-    def registry(cls):
+    def registry(cls):  # pylint: disable=E0213
         return cls._registry
 
     @classmethod
-    def register(cls, object_type, provided, name, factory):
+    def register(
+        cls, object_type, provided, name, factory
+    ):  # pylint: disable=W0613
         pattern = (object_type, provided, factory)
         if pattern not in cls._registry:
             cls._registry.append(pattern)
 
     @classmethod
-    def unregister(cls, object_type, provided, name, factory):
+    def unregister(
+        cls, object_type, provided, name, factory
+    ):  # pylint: disable=W0613
         cls._registry.remove((object_type, provided, factory))
 
-    def __new__(cls, *p, **k):
-        if not '_instance' in cls.__dict__:
-            cls._instance = object.__new__(cls, *p, **k)
+    def __new__(cls, *p, **k):  # pylint: disable=W0613
+        if '_instance' not in cls.__dict__:
+            cls._instance = object.__new__(cls, *p)
```

Comment:
Yes, you are right.  I added back in `**k`, then in next commit removed the `adapt.py` module since it's not being used.